examples/DHS-e2e/README.md

```diff
@@ -1,63 +1,71 @@
-    1. Ensure to update path to input documents in DHS/build.gradle where the task is defined
-5. Run harmonization flow
-  1. gradle hubRunFlow -PentityName=Customer -PflowName=custESJJ
-
-## Consuming curated data from FINAL database ##
-1. cd `<path to DSF>`
-2. Load your APIs into data-hub-MODULES database
-  1. gradle mlLoadModules
-3. Call the API. The API runs a query on FINAL database to return all the Customers who have "Sales" in their title
-  1. gradle runMain
-
-
-___You can verify via REST port 8004 for ingested/harmonized docs___
+# Running Data Hub Service end-end #
+
+There are 2 projects one for each DHS and DSF. DHS is the project to run DHF flows in the stack spun up by DHS. Whereas DSF is the project that consumes curated data in the FINAL database. Commands to be executed in sequence are as under:
+
+## Pre-req ##
+Gradle 4.x+ installed globally.
+* wget https://services.gradle.org/distributions/gradle-4.2.1-bin.zip
+* unzip to a dir of choice
+* update env var PATH in bash_profile
+  * `export PATH=$PATH:<unzipped dir>/gradle4.2.1/bin`
+  * source ~/.bash_profile
+
+There is a gradle task “importAllCustomers” to ingest source documents. So you can either run that task or use your locally installed mlcp.sh as described later.
+* Download and Extract mlcp-9.0.7.zip to a directory of your choice
+  * `export PATH=$PATH:<unzipped dir>/mlcp-9.0.7/bin`
+  * source ~/.bash_profile
+
+
+After creating stack in AWS make sure you have created users with appropriate roles. Update DHS/gradle.properties and DSF/gradle.properties to use them.
+
+* For DHS project
+  * User with flowDeveloper role can create, load into modules DB and run flows
+  * User with flowOperator role can only run flows
+* For DSF project
+  * User with endpointDeveloper role can load into modules DB and call the DSF API
+  * User with endpointUser role can only call the DSF API
+
+
+## Assumptions and things to note ##
+* This project assumes that DHS environment is already provisioned. All the app servers, databases are provisioned and required roles created as described in pre-req above
+* This example was executed from bastion host
+* DHS enables you to configure the endpoints to be private or public. If they are public, you can run this project from your laptop. If the endpoints are private, then these hosts are only accessible from the VPC that is peered to the MarkLogic VPC (This can be accessed from your local by ssh tunneling). In either case please update mlHost in DHS/gradle.properties and DSF/gradle.properties to use Flows endpoint.
+
+
+# Steps to run end to end #
+
+## Getting your flows in DHS ##
+1. cd `<path to DHS>`
+2. Install data-hub core MODULES
+    1. gradle hubInstallModules
+    2. data-hub-MODULES should have 133 documents. You can verify this in your browser:
+        ___http://CURATION_ENDPOINT:8004/v1/search?database=data-hub-MODULES___
+3. Load your modules for input/harmoninzation flows
+    1. gradle mlLoadModules

```

Comment:
```suggestion
    1. Run gradle mlLoadModules
```

Reply:
Its a command so didnt want to include it in a sentence. The sub bullet is the thing to run to get the thing described in main bullet done.